.CI/create_feedstocks

```diff
@@ -10,7 +10,7 @@ if [ -n "$GH_TOKEN" ]; then
     conda config --add channels conda-forge
     conda install --yes --quiet conda-smithy
     # Temporary workaround b/c conda-build-all is not ready for the latest conda-build.
-    conda install -n root --yes --quiet conda-build=1.20.1
+    conda install -n root --yes --quiet conda-build=1.20.3
```

Comment:
This change was actually merged in https://github.com/conda-forge/staged-recipes/pull/979. So you should be able to just rebase your branch.